tests/Functional/tests/standalone-email-addresses/standalone-email-addresses.html

```diff
@@ -80,7 +80,7 @@ <h1>
 <blockquote>
     <hr />
     <dl>
-        <dt> Received <span class="classifier-delimiter">:</span><span class="classifier">from x.y.test</span></dt>
+        <dt>Received: from x.y.test</dt>
```

Comment:
Does this change really belong to this commit?

Reply:
Yes. As said before, this is the absolute crasiest and worst test in this bundle: it's purpose is to test if an emailaddress in a sentence is transformed into a mailto link. It does this by pasting a raw mail message (see `standalone-email-address.rst`) with all headers as being "rst syntax".

This results in this line correctly being parsed as a definition list. The bug however was that we exploded on `":"` to find "definition classifiers". Instead, we should have used `" : "` (as you can see in the bug report from Javier). That means that this crazy raw email message also looses it's classifier.

Reply:
I really mean "commit" here, not PR… your message talks about the relationship between this test and exploding with ` : `, yet test is touched in the commit that removes `{spaceless}`

Reply:
Should it maybe have been touched in both commits?

Reply:
Or should both commits be squashed maybe? It doesn't look right to have a commit with a failing build, then another commit that removes more than just spaces in this test.

Reply:
Oh, I see. Squashed the commits :heavy_check_mark: